src/main/java/com/amazon/opendistroforelasticsearch/sql/esdomain/LocalClusterState.java

```diff
@@ -399,6 +395,12 @@ public FieldMappings(MappingMetaData mappings) {
             fieldMappings = mappings.sourceAsMap();
         }
 
+        public FieldMappings(Map<String, Map<String, Object>> map) {
```

Comment:
Please add documentation for public methods, especially for the cases like this one, where it's not obvious (e.g.  why we need the second constructor).

Reply:
The constructor signature is self-documenting. We are inititalizing a class member using that.

Reply:
will fix the name of parameter from map to mapping